presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveSplit.java

```diff
@@ -55,10 +55,6 @@ public void testJsonRoundTrip()
         AcidInfo.Builder acidInfoBuilder = AcidInfo.builder(new Path("file:///data/fullacid"));
         acidInfoBuilder.addDeleteDelta(new Path("file:///data/fullacid/delete_delta_0000004_0000004_0000"), 4L, 4L, 0);
```

Comment:
if we are keeping delete delta, why are we removing original file? am I missing something?

Reply:
i backed out a change in the file, which seemed unnecessary (test passed with and without).
however, i would be more happy to understand why we want it here.

Reply:
I guess the notion was to check if deserialization of HiveSplit works fine. It may help to catch the errors when some issue with Json serialization and we do have Json params in OriginalFileLocations.

Reply:
I like the idea. It needs to be reflected in the test assertions too. Can you please address this as a follow up PR?

Reply:
yeah, will take this up as a follow up PR.